Rakefile

```diff
@@ -51,3 +51,22 @@ task update_books: :environment do
     end while done == false && i < 5
   end
 end
+
+LEVELS_CONVERT = YAML.load(File.read(File.expand_path('../db/levelsConvert.yml', __FILE__)))
```

Comment:
I think it might be better to move these tasks over to a separate directory, so that this file doesn't get too big and these tasks become more modular. That can be done in a separate PR.

#345